### ISSUE TITLE
clausal appos

### ISSUE THREAD

**Issue by nschneid** [UniversalDependencies/docs#1024]
The [`appos` guidelines](https://universaldependencies.org/u/dep/appos.html) say:

> `appos` is intended to be used between two nominals.

But in treebanks, there are many exceptions where one or both sides of the `appos` is a clause. Here are a few with a VERB:

- [English-EWT](https://universal.grew.fr/?custom=661b0285d767e)
  * The decision was made by ISO upper management with one **goal** in mind: to **keep** the lights on.
  * A quick **question**: we haven't **received** your invoice for the last few model / paper reviews. 
  * **PS** - we also **have** more cats coming in for re-homing see our 'Homes Wanted' page
- [English-GUM](https://universal.grew.fr/?custom=661b05a855a79)
  * more naturalistic experimental **conditions** (e.g., **demonstrating** actions using real objects instead of two-dimensional depictions)
- [Czech-PDT](https://universal.grew.fr/?custom=661b0201a0b3c)
  * Jen jedno nechtěli přiznat - že by štěpný materiál mohl pocházet z Ruska. ("They didn't want to admit just one **thing** - that the fissile material could **come** from Russia.")
- [Hebrew-IAHLTwiki](https://universal.grew.fr/?custom=661b0334ccf50)
  * הדבר מתרחש כתוצאה ממקרי היפוגליקמיה חוזרים ונשנים ולעיתים כתוצאה מתרופות (למשל, חוסמי ביתא מפחיתים את תגובת הרעד ומאיטים את הדופק )
     -  ("This occurs as a result of repeated cases of hypoglycemia and sometimes as a **result** of medication (for example, beta blockers **reduce** the tremor response and slow down the pulse)")

Should these be changed to something else (`acl`/`advcl` for the "for example" clauses, `parataxis`/`discourse` elsewhere)? Or is the definition of `appos` too narrow?

I am assuming that clausal quotations and titles can function as nominals, and thus be valid as `appos`.

**Comment by dan-zeman** [comment]
I think that the current definition of `appos` is too narrow. Like `conj` and `flat`, it should be considered a special relation where we don't distinguish whether the parent and/or the child is nominal or clausal.

Otherwise we would need at least one new relation where a clause is attached as an appositional modifier of a nominal. Saying that it is simply `acl` does not seem right, unless we are ready to say that a NOUN-NOUN apposition is just `nmod`.

**Comment by nschneid** [comment]
> Saying that it is simply `acl` does not seem right, unless we are ready to say that a NOUN-NOUN apposition is just `nmod`.

I have always understood noun-noun appositions as a kind of `nmod` (that happens to have a distinct top-level label, `appos`). Is there some reason not to consider it modification?

**Comment by nschneid** [comment]
From the [2021 CL article](https://doi.org/10.1162/coli_a_00402):

<img width="1072" alt="image" src="https://github.com/UniversalDependencies/docs/assets/985263/d57baee2-bc20-4228-b283-ab226ecf1acb">

**Comment by dan-zeman** [comment]
> > Saying that it is simply `acl` does not seem right, unless we are ready to say that a NOUN-NOUN apposition is just `nmod`.
> 
> I have always understood noun-noun appositions as a kind of `nmod` (that happens to have a distinct top-level label, `appos`). Is there some reason not to consider it modification?

I am not saying it is not modification. But if we think that 1) `appos` is only for nominal dependent on another nominal, as the current guidelines suggest, and 2) `appos` is sufficiently different from `nmod` to warrant a separate relation type, then we lack an analogous distinction in the case where clause is dependent on a nominal: besides `acl`, we would also need `apposcl`, which we don't have.

Alternatively, we could say that `appos` is something special, not necessarily restricted to nominal-nominal modifications (analogously to `conj`, where we also do not distinguish nominals from clauses). I think this was always my preferred understanding of apposition, being influenced by the PDT framework; but it did not match what the UD guidelines were saying. Perhaps because nominal-nominal appositions are the prototypical case, which is much more frequent then the clausal alternatives (and I guess also because we always had many more burning issues to solve), I did not see this as a problem which should be investigated and possibly changed in the guidelines. But now that you've raised the issue, I think we actually do need to resolve it.

**Comment by nschneid** [comment]
I guess my personal view is that apposition is a _semantic_ notion that happens to correlate with syntactic tests like reversibility. So from a theoretical perspective it is a bit odd to single it out from other kinds of modification. From a practical perspective, the noun-noun variety is frequent enough in many genres that it is handy to have a distinct `appos` label, but it seems we need to clarify what to do with the clausal ones.

Do we agree that "conditions (e.g., demonstrating actions...)" and the other exemplification ones should not be `appos` even under an expanded definition, because they are not reversible?

**Comment by dan-zeman** [comment]
> Do we agree that "conditions (e.g., demonstrating actions...)" and the other exemplification ones should not be `appos` even under an expanded definition, because they are not reversible?

For me, reversibility is not a requirement. The Czech example you found is not exemplification but it is not reversible either. I am almost sure that exemplifications are solved as `appos` in the Prague data, too.

**Comment by amir-zeldes** [comment]
I think there are lots of cases of 'conversion', where a clause is effectively treated as a nominal. A case in point is verbal compound modifiers in languages like English, where we have handled the external dependency as `compound` and the internal dependencies compositionally based on the argument structures of the nested predicate.

From that perspective, I think examples like:

  * A quick question: we haven't received your invoice 

Seem legitimate to me, and these are arguably not 100% irreversible if we think of examples like:

  * I asked you: "did you do it?" - a simple question.
  * I asked you a simple question - "did you do it?"

That said, I think examples with parentheses are sometimes ambiguous between such an apposition and generic parenthetical parataxis, where a clearly non-appositive clause could stand just as well. I'm not completely happy making this distinction based on orthography, but I think things like:

  * more naturalistic experimental conditions (e.g., demonstrating actions using real objects instead of two-dimensional depictions)

should probably be parataxis, and indeed they are not reversible. I'm also not sure I want reversibility to be a totally necessary criterion for each exemplar, but if we are looking at a class of expressions and they are all non-reversible, it seems a bit suspicious to me.

**Comment by amir-zeldes** [comment]
>  my personal view is that apposition is a semantic notion

PS - I know we see these things a little differently, but apposition behaves differently around pronominalization and other types of anaphora, since it does not introduce an additional discourse referent. For me this has a semantic aspect of course, but also a morphosyntactic one in terms of binding, coordination, number agreement and other issues that have syntactic reflexes, so I wouldn't say it's a purely semantic notion.

**Comment by nschneid** [comment]
Considering noun-noun appositions, I take it we would use `appos` for (1) and (2) but not (3):

1. Giant, a/the supermarket chain, reported quarterly earnings...
2. Going to Giant (big supermarket)
3. Going to Giant (big sale)

The `appos` guidelines give some criteria:

> Good tests include to ask whether the two halves are full nominals, whether the two halves can be swapped or not, and whether there is case or agreement concord (in a language with rich morphology).

All I am saying is that, to a first approximation, (1-3) are all nominal parenthetical modifiers of a nominal, and the distinction between (2) and (3) seems pretty minor as a matter of grammatical function. I should have restricted my statement above to English though—in a language with richer morphology there may be a stronger case to distinguish them (if the apposition construction involves special morphosyntax not following from general principles of referring expressions in the language).

**Comment by sylvainkahane** [comment]
Following Blanche-Benveniste's grid analysis (Blanche-Benveniste et al. 1979), we decided, in the Rhapsodie treebank (Gerdes et Kahane 2009; Kahane et al. 2019) and then in SUD, to consider different types of paradigmatic constructions all annotated as subrelations of `conj`:

- `conj:coord` for what is annotated `conj` in UD;
- `conj:appos` for what is annotated `appos` in UD;
- `conj:dicto` for what is annotated `reparandum` in UD (but reversed).

The reasons of this choice, well documented in different papers, in that in all these cases, we have conjuncts, that is, constituents that are in a paradigmatic relation, that is, that can replace each other (they are also in a syntagmatic relations and are uttered after each other). The syntax of all these phenomena is similar, but semantics is different, and the markers are different:

- for `conj:coord`, the two conjuncts denotes different referents and the markers are coordinating conjunctions;
- for `conj:appos`, the two conjuncts are two denotations of the same referent; they are markers such as _namely_ or _that is_; 
- for `conj:dicto`, the two conjuncts elaborate a same denotation; there are markers such as _uh_.

[Here](https://universal.grew.fr/?custom=661ce985ad97b) are our annotations for spoken French. Let me recall that Grew-match allows you to see in the same window the SUD and the UD annotation.
[Here](https://universal.grew.fr/?custom=661cea222f4ef) are our annotations for spoken Naija, an English-lexified pidgincreole.

Paradigmatic relations are not restricted to nouns and you can see that `conj:appos` is used for all POS: [French](https://universal.grew.fr/?custom=661ceb08ad672), [Naija](https://universal.grew.fr/?custom=661ceac7f19ea).

A last remark: we also consider that UD `appos` is used for two different constructions: one is the paradigmatic construction `conj:appos` and the other is  a modificative construction, that we annotate `mod@appos`; [Fench](https://universal.grew.fr/?custom=661cecc93c252). The `@` means that we do not consider `mod@appos` to be be a syntactic subrelation of `mod`, but just a particular case of `mod` (`mod` is the SUD relation that subsumes UD `nmod, amod, advmod, acl, advcl`…). The situation is different for `conj:appos`, which is considered as a different but similar construction to `conj:coord`.

Blanche-Benveniste, C., Borel, B., Deulofeu, J., Durand, J., Giacomi, A., & Loufrani, C. (1979). Des grilles pour le français parlé. _Recherches sur le Français Parlé_, Aix-en-Provence, (2), 163-206.

Gerdes K., Kahane S. (2009) [Speaking in piles: Paradigmatic annotation of French spoken corpus](https://kahane.fr/wp-content/uploads/2017/01/piles-liverpool2009.pdf), _Processing of the fifth Corpus Linguistics Conference_, Liverpool, 15 p.

S. Kahane, P. Pietrandrea, K. Gerdes (2019) [The annotation of list structures](https://kahane.fr/wp-content/uploads/2021/07/rhapsodie-5-lists.pdf), in Lacheret-Dujour A., Kahane S., Pietrandrea P. (eds) (2019), _Rhapsodie – A Prosodic and Syntactic Treebank for Spoken French_, John Benjamins, Amsterdam, 69-95.

**Comment by amir-zeldes** [comment]
> the distinction between (2) and (3) seems pretty minor as a matter of grammatical function

I don't think I agree with that, for argument structure reasons, though I suspect @nschneid might see these as purely semantic ground. Appositions do not add another participant to argument structures: If we meet "Kim Wang, the lawyer", we meet only one person (a semantic fact), but also syntactically, only one argument is saturated (object of meet), and in passivization, we see the expected equivalent "Kim Wang, the layer, was met by..."

This is not true of "big sale". It's true that the grammatical function of this phrase is underspecified in the example due to the parenthetical structure, which is why we assign `parataxis` as the deprel. But whatever the underlying semantic role (cause: we went to Giant *because* of the sale; temporal - *during* a sale; etc.), it is a distinct role filler from "Giant". This has syntactic reflexes too - I can say:

  * Going to Giant (big sale) later today
  * Going to Giant later today (big sale)
  * Today (big sale) we're going to Giant

etc. This is generally not true of "big supermarket"; to preempt the inevitable counter-example, and at the risk of being circular, I will say that if "big supermarket" appeared with an intervening phrase separating it from Giant, I would be inclined to see that as `parataxis` as well. In other words, I think appositions create phrase nodes with their heads, and parataxis does not.

> The syntax of all these phenomena is similar, but semantics is different, and the markers are different

I think it is reasonably distinct: coordination can create plurals out of multiple singulars, but apposition cannot, for example. Reparanda are something totally different. They can happen mid phrase and be repaired by a totally unlike phrase, whereas coordination typically takes like-conjuncts (admittedly not always), and generally forms a valid phrase that can saturate argument structure. The repaired part in reparanda is not generally part of the argument filling operation of the head of the repair. Again, some might want to see this as a purely semantic difference, but I think argument filling has a very form-based syntactic side - for example in a repaired subject, on the gender and number of the repair determine agreement with the verb. The reparandum is irrelevant, but this is not true in coordination.

**Comment by nschneid** [comment]
Looking at this again I realized that (3) may have multiple interpretations. Let me go with a more elaborate example:

(3') Going to Giant (big sale) and not Safeway (too expensive).

Is "to Giant (big sale) and not Safeway (too expensive)" a constituent functioning as a goal oblique? Arguably there's ellipsis going on, but I'm not sure if it's a kind of ellipsis that would be made explicit in UD.

**Comment by Stormur** [comment]
The definition of `appos` in the guidelines is very clear, and it even says the apposition has to follow immediately. I cannot retrieve an old discussion where this was debated, but there in the end it was clarified that, in fact, `appos` is simply a particular type of `nmod`. So one wonders why this relation is not used instead. 

But actually, in many cases what comes out is that "appositions" are rather flat structures, so probably `flat` is the best relation. This without requiring any particular direction or "reversibility", at least not from a semantic point of view, since nothing really is reversible at that level. 

> I think it is reasonably distinct: coordination can create plurals out of multiple singulars, but apposition cannot, for example.

For example, I do not think this is really relevant, since agreements of these kinds can vary even with simplex subjects, when semantics overrides morphology.

In the Latin treebanks these appositional constructions are very widespread and have posed difficulties from the beginnings, also with regard to the conversion from other annotation formalisms. In the [presentation of the Index Thomisticus (2018)](https://aclanthology.org/W18-6004.pdf) this was discussed, and an admittedly wacky solution of using a subtype `appos` with relations such as `acl`, `nmod`, `advcl`... was introduced. But then, at least from [v2.6 and the LLCT](https://aclanthology.org/2020.lrec-1.117.pdf) we started using `conj:expl` ([documented here](https://universaldependencies.org/la/dep/conj-expl.html)), which seems to correspond to SUD's `conj:appos`. Here `expl` stays for explicative, since these are more or less "that is"-expansions. Anyway, I would prefer not using subtypes which have the same name as existing relations.

I second all the intuitions that converge on a co-ordinational treatment of similar expansions. I do not think that the semantic level of having or not the same referent should be reflected on the main relation type, as far as the structure is really the same of co-ordinations, but the subtype is there exactly for this reason (while the specification of a co-reference is a different annotation level of its own). This is also seen in the symmetry of the elements: even when there is a clausal element expanding a nominal one (and there are many in the IT-TB), it is always a nominal form (such as eng. _-ing_) or is introduced by a marker, also used for `ccomp` etc... and so on for all other parts of speech. 

I also agree with the distinction from and preference for `parataxis` in some cases, as explained by @amir-zeldes : parentheticals are another phenomenon. What is labelled as `parataxis` and attached to another node of the tree is actually a different, "interpolated" syntactic tree (elliptical or not) which might as well have its own, non connected root. 


> * A quick **question**: we haven't **received** your invoice for the last few model / paper reviews.

This looks very much like a `parataxis`, or, maybe better, a topic-comment construction, so _a quick question_ could be `dislocated` with respect to the main clausal component _we haven't..._

---

To sum it up, in my opinion:

* `appos` does not really look justified, and could be covered by `nmod` or even `flat`;
    * here the term "apposition" seems to actually be used for "epithet" 
* appositions in a broader, more conventional sense are a kind of co-ordination (they very often use specialised connectors, or are asyndetic the same way as other co-ordinations), and so they could be annotated as a subtype of `conj`.

Again, it seems here that a very narrow, idiosyncratic interpretation in the UD guidelines of a general term which usually has a much broader connotation is creating misunderstandings and confusions.

**Comment by jnivre** [comment]
I have always thought of "appos" as a semantically defined subtype of "nmod", motivated by the original IE-type applications of Stanford Dependencies (where it is useful to know that the two nominals have the same referent). If we were designing the relation system now, I would argue that "appos" is not needed or should at most be a subtype.

**Comment by amir-zeldes** [comment]
> I have always thought of "appos" as a semantically defined subtype of "nmod"

I disagree, and I don't think I've heard a convincing argument why apposition is not a syntactic phenomenon given its behavior around argument structure and morphological agreement, neither of which is a purely semantic construct. For me, nmod means that we take one phrase to modify another, separate and distinct phrase, creating a nested phrase structure. As with phrases in general, we can use movement, pronominalization, coordination and interrogation to recognize that the modifier is such a phrase. But apposition does not work the same way:

  * The King of Spain - `nmod(King, Spain)`
  * Pronominalization: The King of it (it=Spain)
  * Coordination: The King of [Spain and Portugal] (e.g. Philip II was King of both)
  * Interrogation: The Kind of what? Spain
  * Movement: Of Spain there was no greater King than Philip II (disclaimer: not saying this is true)

Apposition:

  * The King, Philip II - `appos(King, Philip)`
  * Pronominalization - ?The King, he (the opposite works better admittedly)
  * Coordination - *The King, Philip II and Juan Carlos (this may seem odd to test, but if it were an `nmod`, coordination should not be prohibited inside the modifier)
  * Interrogation: *The King, what/who? (I think "King who?" is fine, but that's a different construction)
  * Movement: *Philip II, there was no greater King than

I think the differences above are syntactic, and apposition is commonly treated as a syntactic phenomenon and described in most grammars under the syntax of noun phrases (incl. by CGEL for English BTW)

**Comment by nschneid** [comment]
> apposition is commonly treated as a syntactic phenomenon and described in most grammars under the syntax of noun phrases (incl. by CGEL for English BTW)

Apposition is certainly worth discussing in a reference grammar as there are certain conventions for elaborating on one nominal with a second nominal whose referent is the same. I think the notion of apposition crucially depends on meaning (in that there has to be an elaboration of a shared referent in a particular instance, not just in prototypical instances). A particular apposition construction will have some morphosyntactic particularities, and maybe there are even some universal morphosyntactic properties that hold—but is this reason enough to consider it outside the realm of nmod etc.?

Of course, "the city of Paris" is *not* what we are calling apposition, despite the sharing of a referent. That is obviously a PP modification construction (nmod). The question is whether the top-level grammatical function per UD principles is necessarily different, not whether they are different constructions at some level of granularity.

CGEL: "Appositive modifiers: Appositive dependents are ones which when substituted for the matrix NP in a declarative clause systematically yield a clause which is an entailment of the original [...e.g.] _the opera **'Carmen'**_". And: "The construction with a specifying NP as supplement is known as apposition. More particularly, this is the supplementary type of apposition [...e.g.] _A university lecturer, **Dr. Brown**, was..._"

This is basically saying that there are two types of appositive constructions, one of which is a modification construction inside an NP, and one of which is a supplement (~parenthetical), which is a somewhat looser kind of relation than modifier. Note the definition in terms of entailment—so the "apposition" designation cross-cuts grammatical function (modifier vs. supplement).

UD doesn't exactly recognize a notion of supplement/parenthetical—it has `parataxis` as a way to handle some of them, but prefers `acl:relcl` for supplementary relative clauses. It occurs to me that most nominal parentheticals are probably appositions, so the label `appos` has freed us from having to debate `nmod` vs. `parataxis`. But not all `appos` uses are parenthetical ("the opera 'Carmen'" isn't).

Turning to your tests: I would surmise that there are discourse factors that weigh against a pronominal appositive (interrogative or otherwise) or movement in most circumstances, because an apposition is about adding elaboration of some entity that has been invoked. Coordination is fine as long as the nominals can be construed as referring to the same group: "The authors, Amir and Nathan, claimed...".

If your point is that appositives can appear in places that most nmods can't, and can't do all the things that other nmods can, then yes, I agree. That could be a reason for a subtype (cf. English `nmod:npmod`, which has distinctive properties from plain `nmod` i.e. PPs). But the principal ones seem to meet the basic criterion for what UD calls an `nmod`, i.e. that there is a full nominal modifying another nominal.

**Comment by amir-zeldes** [comment]
> If your point is that appositives can appear in places that most nmods can't, and can't do all the things that other nmods can, then yes, I agree. That could be a reason for a subtype (cf. English nmod:npmod, which has distinctive properties from plain nmod i.e. PPs)

I don't think that rationale discriminates between subtypes and major types. The only difference that I am aware of between advcl and ccomp is that they appear in different places and can't do the same things. The only other type of criterion for deprels is ontological, i.e. regarding the morphosyntactic kind (phrase/pos), which is why English `compound` != `amod`, despite the fact that these mostly can stand in the same positions (but they are ontologically different at the category level). Since advcl and ccomp have the same category (clause), they are both effectively only distinguishable by how you can use them. 

All things being equal, I prefer not to rock the boat: `appos` has been with us for a very long time and describes what is in most cases a recognizable and cross-linguistically very well attested grammatical function. I would not want to see it go away just because in some fairly rare circumstances it has fuzzy boundaries with another category - this is true of a wide range of distinctions in UD.

**Comment by nschneid** [comment]
Yeah, we're not going to eliminate `appos` in the near future, so the question is: should we understand it as a carve-out from `nmod`, as the current guidelines say, and use another relation for dependents that are not nominals reversible with the head nominal? Or should `appos` be broadened to include non-nominal dependents that explicate a nominal (e.g. "we have one goal in mind: to keep the lights on")? Or followup explications not limited by the category of either the head or the dependent? We would have to explain how to tell the difference between `appos` and `acl`/`parataxis`/`conj`.

**Comment by amir-zeldes** [comment]
I would be in favor of allowing `appos` for "we have one goal in mind: to keep the lights on", for the same reasons I think a verbal modifier in a compound should retain the external deprel `compound` in cases like "an I don't care/compound attitude". The internal deprels reflect the compositional structure up to the point of attachment to the nominal, and that's all wrapped in what constituent syntax could interpret as a unary conversion of the type [[I don't care]S]NP

**Comment by nschneid** [comment]
So how would you describe the characteristic of that dependent that makes it `appos`? What about:

- We have one goal in mind: We should keep the lights on.
- The parties have several goals: first, they want to do X; second, they want to do Y.
- We are preparing for dinner - cleaning the house, cooking the food, and setting the table.

**Comment by amir-zeldes** [comment]
As mentioned above, I completely agree that there are fuzzy corner cases we'd have to make decisions on, just like with the contact points between other deprels and the periphery of their usages. Concretely for the first case, my preference would be to use `parataxis` if neither of the constituents is a nominal. For the second case, if we split them intro several sentences then there are no deprels between them anyway.

I see you edited it to have semi-colons, but even in that case, I think the relationship between the 'first' clause and the 'second' clause is parataxis, so there is no single phrase in the `appos` relation to the NP, so again it's parataxis. But realistically, I think such an example in a single sentence would be very rare, so I wouldn't worry about it too much.

**Comment by amir-zeldes** [comment]
Oh wait,  I see you edited it again, it's hard to keep up :)

For the third example which currently reads "We are preparing for dinner - cleaning the house, cooking the food, and setting the table" I would say it's not `appos` due to the "no nominal" principle. But this would be `appos` under my proposal:

  * We are making three preparations - cleaning the house, cooking the food and setting the table.

Because preparations is a nominal and the expansions are coordinated into a single phrase (no parataxis). I'm not saying there is no other way to see it, but I think that's a fairly usable guidelines, and again, I predict this will come up rather rarely.

**Comment by nschneid** [comment]
- I am editing my book - the introduction, the conclusion, and the acknowledgments.

(Trying to see whether you think the head needs to be nominal)

**Comment by nschneid** [comment]
- Controlling the bow - preparation, attack, release - are crucial to producing a good sound.
   * or, "Making sure to control the bow"

**Comment by amir-zeldes** [comment]
> I am editing my book - the introduction, the conclusion, and the acknowledgments. (Trying to see whether you think the head needs to be nominal)

I don't follow - here both the head and the expansion are nominal, just like in a standard apposition, no?

> Controlling the bow - preparation, attack, release - are crucial to producing a good sound. 

Here the head is a non-nominal, and what's more, agreement suggests that the verb actually agrees with either the expansion, or a coordination of all four heads (controlling the bow + prep + attack + release). Even if we wanted to accept appositions where the initial head is non-nominal (which I think would be a slippery slope), this would not be a case of that, because then I would expect singular "is".

So I guess in sum, among other reasons to ensure consistency, I would require the head to be a nominal, but relax the requirement on the dependent in cases where we can interpret the modifier as a nominalized clause. I believe this is analogous to what we're doing with phrase-modifier compounds.

**Comment by nschneid** [comment]
Oh, yeah, I should have written "is" instead of "are".

What about: `appos` requires (1) a nominal head and (2) a nominal or non-finite clause dependent? Can you think of ways you'd want to broaden it beyond that?

**Comment by nschneid** [comment]
Sorry, just noticed the Czech example above is finite. Make that:

`appos` requires (1) a nominal head and (2) a nominal or non-relative subordinate clause dependent?

**Comment by amir-zeldes** [comment]
> a nominal or non-relative subordinate clause dependent?

I guess that depends on what you consider to be subordinate - I would be OK with a regular looking sentence, which would only be subordinate in the sense that it's nested into an NP:

  * That event, Russia launched Sputnik before the US had any satellites, changed the world.

I also think this would sound much better in a non-finite version (Russia launching), but if this sentence appeared in data somewhere, I'd be OK with considering it an apposition despite being 'main-like' (but subordinate in being inside the subject NP, maybe almost like a quotation cast as an NP).

**Comment by nschneid** [comment]
A few from browsing [adnominal `parataxis` in EWT](https://universal.grew.fr/?custom=6622bc03a3068):

- However, do not purchase tickets until it is a done deal - a **lesson** I learned following the Ark game last season.
   * not nominal-headed, so keep parataxis
- I just finish reading a really good book (it was actually a **ebook**) from some new series called the The Tale of Terra.
   * re-articulation of a noun label with a finite clause (parenthetical, non-subordinate)
- Superior work - always **comes** through when we need him. 
   * expansion with an "in that..." meaning
- Moral of the story: Don't **drink** Coke..........drink Pepsi!
   * imperative part is essentially a quotation
- The best Thai food I've ever had in Australia, very **fresh** and so much favor of authentic Thai .
   * I assume this would stay parataxis because the dependent is headed by "fresh" meant as a predicate, so there is no coreference—but what if it were "very fresh food"?

Should any of the noun-headed ones be excluded from a broader definition of `appos`?

**Comment by amir-zeldes** [comment]
I think 4. is supposed to be appos even using the old ":" as an equative (key-value style) guideline. Otherwise it would be nsubj headed by "drink" (=the moral is don't drink). So yeah, that and possibly 3 would be appos (it's possible there are cases that are genuinely ambiguous between parataxis and appos, and we don't know what the speaker intends). 

For 5 I think it's not appos, but if it were "very fresh food" it would be a candidate if that's how it's intended. But it would be ambiguous with an elliptical subjectless clause: (it was) the best food I had in Australia, (it was) very fresh food. Maybe the reason I'm a bit suspicious is the short-before-long tendency: if it were intended as an apposition, I would expect: "very fresh food, the best food I had in Australia". But for a slam dunk you would need the whole thing to serve in a role slot of something else, which we don't have here since it's a fragment. The following is IMO unambiguous:

  * Very fresh food, the best food I had in Australia, was the Pad Thai on the beach at Ko Samui.

Because there is only 1 nsubj slot, it's clear it's saturated by this one NP. But this issue is not actually related to the nominal vs. verbal thing we're discussing, it applies equally to two nominals where it's unclear if it's two fragments in parataxis (esp. if each is implicitly predicative with zero copula) or an apposition and just one holistic NP.

**Comment by nschneid** [comment]
What about (2), an aside that comments on a nominal? "a really good book (it was actually a ebook)"

> I think 4. is supposed to be appos even using the old ":" as an equative (key-value style) guideline.

Dunno about that key-value policy—I take that guideline to be about structured data rather than real sentences: "appos is also used to link key-value pairs **in addresses, signature blocs, etc.** (see also the `list` label)"

It seems to me that "Moral of the story:" is a syntactically reduced way to convey a semantic predication, not modification as in canonical `appos`. The relation between the parts feels discursive, somewhat similar to the "X, so Y" construction, but more of a topic-comment structure as pointed out by @Stormur. Notice that there's no determiner before "moral" either.

"Moral of the story: Don't drink it" feels an awful lot like "Status of the baby: sleeping/asleep". Which brings us to things like

- The baby, **sleeping/asleep** in his crib, made no sound.

We don't want to call that `appos` right? Is it `acl`?

**Comment by sylvainkahane** [comment]
I give you some examples of things we annotated with `appos` in the French-Rhapsodie treebank of spoken French.
The table with all cases is here: https://universal.grew.fr/?corpus=UD_French-Rhapsodie%402.13&table=yes (just click on appos in the left column)

NOUN -[appos]-> VERB
- et à la fin de sa vie, euh, quand il est en France pour soigner sa tuberculose
and at the end of his life, uh, when he's in France to treat his tuberculos
- son message initial au tout début de l'évangile, convertissez-vous, le royaume de Dieu est proche de vous, son message s'inscrit aujourd'hui dans sa personne. 
his initial message at the very beginning of the gospel, Convert, the kingdom of God is near you, his message is now inscribed in his person.

NUM -[appos]-> VERB
- en mille neuf cent cinquante, quand vous avez fait paraître ce chef-d'œuvre
in one thousand nine hundred and fifty, when you published this masterpiece

NOUN -[appos]-> ADJ
- donc, là, vous partez dans cette direction, toujours tout droit.
so here you go in this direction, always straight ahead.

ADV -[appos]-> NOUN
- il y a deux collèges là, rue d'Alésia.
- there are two middle schools here, on rue d'Alésia

VERB -[appos]-> VERB
borderline case: we can consider there is an appos between the subordinate clauses because they refer to the same  fact (what I understood), they are reformulations of each other and it will be strange to add a CCONJ 'and' here:
- j'ai compris que ça allait, que vous étiez content, que il y avait suffisamment de, d'espaces verts, de, d'équipements et caetera.
I understood that things were going well, that you were happy, that there were enough green spaces, amenities and so on.

**Comment by sylvainkahane** [comment]
About the minimal pair proposed by @nschneid:

1. We have one goal in mind: to keep the lights on.
2. We have one goal in mind: We should keep the lights on.

When you annotate spoken data, you don't have any punctuation. One very important criterion is to decide what can be an independent utterance (we call this an illocutionary unit). In (1), "to keep  the lights on" is not an illocutionary unit and it can commute with "one goal"; it why we would have chosen `appos`. Not in (2), where we would be likely to have consider two illocutionary units.

**Comment by nschneid** [comment]
More data from EWT, where the "expansion" interpretation of appos creates nonprojectivity:

- Now that Afghanistan, Iraq and Libya are out, two and a half terrorist states remain: Iran, Syria and Lebanon, the latter being a Syrian colony.
  * currently appos(states, Iran)
- The decision was made by ISO upper management with one goal in mind: to keep the lights on. 
  * currently appos(goal, keep)

**Comment by amir-zeldes** [comment]
I would be inclined to view these more as parataxis. The second one is totally not reversible, and the first isn't IMO if we keep "the latter". It's really a complete sentence followed by an afterthought or elucidation. 

The only really convincing exception to adjacency that I've seen for appos is when you get totally a-syntactic things, like a Wackernagel particle assuming second position with total disregard to the surrounding phrase structure. Since those kinds of things could appear in the middle of an NP anyway, they don't form an argument against phrasehood, and the whole thing is still an apposition.

**Comment by Stormur** [comment]
Or maybe bette `dislocated`r: these blocks are really just copies/expansions of other arguments, I see no clausal structure implied. 

The attachment is an issue, because they do seem tied to the "original" argument, but maybe this is something for another annotation layer (co-reference?) and we can just be content with the dislocation attachment to the root to avoid systematic nonprojectivity?

Another issue are some particles and expressions which regularly introduce these blocks, sometimes specialised. Are this indeed a hint to a tighter cohesion than `dislocated`/`parataxis`? A sort of subordination maybe? Maybe not, they are just needed for sentence coherence and the only thing they tell us is that such blocks do have to be connected to the main clause.
